src/main/cpp/Robot.cpp

```diff
@@ -55,6 +56,7 @@ void Robot::DisabledPeriodic() { }
 void Robot::AutonomousInit() {
     drivetrain.resetState();
     elevarm.resetState();
+    led.resetState();
```

Comment:
led.resetState() should be called on init of the subsystem, so this does not need to be done